tectonic/XeTeXFontInst.cpp

```diff
@@ -315,6 +315,9 @@ XeTeXFontInst::initialize(const char* pathname, int index, int &status)
     TT_OS2* os2Table;
     FT_Error error;
     hb_face_t *hbFace;
+    size_t sz;
+    ssize_t r;
+    FT_Byte *data;
```

Comment:
Late-time variable declarations are legal in C++, I believe, and they're kind of nice, so I would rather leave them where they were before in this function.